http/src/main/resources/reference.conf

```diff
@@ -439,6 +439,7 @@ clusterToolMonitor {
 
 leoExecutionMode {
   backLeo = true
+  frontLeo = true
```

Comment:
this is the real fix...in fiab, since we're running both backleo and frontleo, previous dateAccessedUpdater was only run on frontleo instances, which causes fiab to not run dateAccessedUpdater

Reply:
I see.. I guess we have 3 cases:
* backLeo = true
* frontLeo = true
* backLeo = true and frontLeo = true (the fiab case)

I guess another way to define this would be an enum with 3 values: BACK, FRONT, COMBINED.

But I don't feel strongly, this is probably fine.

Reply:
yea, enum might be better...I'll give it a try

Reply:
updated https://github.com/broadinstitute/firecloud-develop/pull/2183 to use enum

Reply:
@rtitle updated https://github.com/broadinstitute/firecloud-develop/pull/2183 to support enum